src/com/base/game/dialogue/places/dominion/shoppingArcade/SuccubisSecrets.java

```diff
@@ -37,7 +37,7 @@
 
 		@Override
 		public String getContent() {
-			if(Main.game.getDialogueFlags().kateIntroduced) {
+			if(!Main.game.getDialogueFlags().kateIntroduced) {
```

Comment:
This adds a bug where you don't know about kate yet and yet the game mentions "you wonder if you should pay Kate another visit"